### DISCUSSION TITLE
CheckUpdates doesnt work

### DISCUSSION THREAD

**Discussion by DasLixou** [qtile/qtile#3784] [Q&A] [answered]
my checkupdates logs to the xserver that the command checkupdates is not found. for distro i use `Arch`

**Comment by elParaguayo** [comment, marked as answer]
The command `checkupdates` is installed as part of the `pacman-contrib` package.

**Reply by DasLixou** [reply]
Ok got it working, i had to install it and had to change Distro to `Arch_checkupdates`. You should include that to the docs :)

**Reply by elParaguayo** [reply]
Yes - available options should be listed. Need to fix that.
